operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/filter/EventFilter.java

```diff
@@ -0,0 +1,22 @@
+package io.javaoperatorsdk.operator.processing.event.source.filter;
+
+public interface EventFilter<R> {
+  @SuppressWarnings("rawtypes")
+  EventFilter ACCEPTS_ALL = new EventFilter() {};
+
+  default boolean acceptsAdding(R resource) {
+    return true;
+  }
+
+  default boolean acceptsUpdating(R from, R to) {
+    return true;
+  }
+
+  default boolean acceptsDeleting(R resource) {
+    return true;
+  }
+
+  default boolean rejects(R resource) {
```

Comment:
Why is this method works with inverted logic?

Reply:
Because I think it is more explicit that way because the generic filter is about overriding whatever the other filters say, rejecting resources even if they would be accepted by the other filtering methods…

Reply:
I think that brings more harm then good, imho even goind with this one interface, that should not be inverted. It is quite straigforwad that there is a conjuction between those rules.